mos-platform/atari2600-common/mapper_macros.h

```diff
@@ -0,0 +1,24 @@
+#ifndef _MAPPER_MACROS_H
+#define _MAPPER_MACROS_H
+
+#include <mapper.h>
+
+#define MAPPER_CART_ROM_KB(kb) \
+    asm(".globl __cart_rom_size\n__cart_rom_size = " #kb)
+
+#define _STRINGIFY(x) #x
+#define STRINGIFY(x) _STRINGIFY(x)
+
+#ifdef MAPPER_BANKED_ROM
+// section attribute doesn't seem to recognize C "concatenated" "strings"
+#define _CODE_BANK(index) __attribute__((section(".rom" index ".code")))
```

Comment:
Can you provide more detail about what the problem is here? This section of macros seems very strange to me, similarly the distinction between DECLARE_XRAM_VARIABLE and _DECLARE... doens't seem to pull any weight.

Reply:
Nvm, figured it out (I think).